engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java

```diff
@@ -25,6 +25,8 @@
 @ApplicationScoped
 public class TemplateService {
 
+    public static final String HCC_LOGO_TARGET = "Logo-Red_Hat-Hybrid_Cloud_Console-A-Reverse-RGB";
```

Comment:
Same as the others - lets move this to tests.

Reply:
Sure, I did it